zap/src/main/java/org/parosproxy/paros/core/scanner/Alert.java

```diff
@@ -1087,6 +1089,59 @@ public Builder setTags(Map<String, String> tags) {
             return this;
         }
 
+        /**
+         * Adds an Alert tag with the given key (tag/name) to the existing collection for this
+         * Alert/Builder.
+         *
+         * @since 2.12.0
+         */
+        public Builder addTag(String tag) {
```

Comment:
These new methods need to be exposed in the `AlertBuilder`s, otherwise the chaining will not work properly (to raise the alert).

Reply:
K